apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,71 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col h-[calc(100vh-6rem)] justify-between content-start gap-9">
       <div>
```

Comment:
On mobile, there should be an Icon Button to bring out the Navigation Rail. If you haven't made the Icon Button component, see [its specifications here](https://m3.material.io/components/icon-buttons/specs).

Reply:
I would be working on a bunch of rework very soon so I think that would be kept for next pr when I actually implement components